packages/surveys/src/components/wrappers/stacked-card.tsx

```
@@ -0,0 +1,122 @@
+
+    if (offset === 0) {
+      setContentOpacity(0);
+      setTimeout(() => {
+        setContentOpacity(1);
+      }, 300);
+    }
+  }, [offset]);
+
+  return (
+    <div
+      ref={(el) => (cardRefs.current[dynamicQuestionIndex] = el)}
```

Biome (1.9.4)

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)